server/routes/notificationRouter.ts

```diff
@@ -0,0 +1,28 @@
+import type { NextFunction, Request, RequestHandler, Response, Router } from 'express'
+
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import NotificationService from '../services/notificationService'
+
+const oneYearMs = 52 * 24 * 3600 * 1000
```

Comment:
this is only 52 days

Reply:
https://github.com/ministryofjustice/hmpps-incentives-ui/pull/362/commits/62c47043f7c1e9951f05992842573f1ba19cac62